credits.txt

```diff
@@ -157,63 +164,69 @@ contributed to Endless Sky:
   DeBlister
   DingusShingleton
   Disiuze
+  DJF113
+  DownsB
   dplepage
+  dragonmaus
   DrBlight
   dreness
   dufferzafar
   dzhu
   eflyon
-  EjoThims
   elgeonmb
   Eloraam
   Elyssaen
-  endless-sky
   EndrosG
+  EricFromCanada
   fcfort
   Ferociousfeind
-  Floppa-Priest
   finite-galaxy
   FixItYondu
   flaviojs
+  Floppa-Priest
   FranchuFranchu
   Fzzr
   Galaucus
   gecko-locator
-  Gods-Righthand
   guillochon
   gunqqer
   Hacklin
   Hadron1776
   har9862
+  Hurleveur
   Hyugat
   InfiniteDonuts
   infinitewarp
   ItsNickBarry
   jafdy
+  Jamibaraki
   Janaszar
+  JanErikAhrens
   janisozaur
-  jarekchr
```

Comment:
I can't find any contributions for this user either. So maybe keep it?

Reply:
#5911